selfdrive/selfdrived/selfdrived.py

```diff
@@ -270,6 +270,10 @@ def update_events(self, CS):
     if not REPLAY and self.rk.lagging:
       self.events.add(EventName.selfdrivedLagging)
     if len(self.sm['radarState'].radarErrors) or ((not self.rk.lagging or REPLAY) and not self.sm.all_checks(['radarState'])):
+      if "unavailableTemporary" in self.sm['radarState'].radarErrors:
+        self.events.add(EventName.radarUnavailableTemporary)
+      else:
+        self.events.add(EventName.radarFault)
```

Comment:
one we split these out, we can use the normal CAN error alert for the radar CAN error too